### ISSUE TITLE
Add Waiting page ?

### ISSUE THREAD

**Issue by JamesDAdams** [adyanth/traefik-container-manager#11]
It would be great as a feature to be able to define a waiting page while waiting for the service to start, if we have docker containers that take time to start we could have a waiting page that appears that we could customize and once the container is loaded we are redirected. 😀

**Comment by adyanth** [comment]
Yeah, that thought indeed crossed my mind. It was not very straightforward since there was no way to check the same from the middleware itself (since the app behind can also give 404/502s even when up causing an infinite loop). 

The best way I can think of would be for the middleware to send a non standard HTTP status code, which can be handled by Traefik's error handler option to be sent to the manager, who can serve a static page with JS to reload the page every 5 seconds or something.

**Comment by JamesDAdams** [comment]
ok

**Comment by JamesDAdams** [comment]
maybe he can help https://github.com/guillaumebriday/traefik-custom-error-pages

**Comment by modem7** [comment]
> maybe he can help https://github.com/guillaumebriday/traefik-custom-error-pages

That project is now deprecated, replaced with https://github.com/tarampampam/error-pages

This feature request would definitely be useful though. 

These guys seem to have cracked the loading page for a very similar project: https://github.com/acouvreur/traefik-ondemand-plugin
